### ISSUE TITLE
annoyed at keras

### ISSUE THREAD

**Issue by hockman1** [keras-team/keras#14319]
Anyone feels that keras is limited to only basic operations and you cant really do much if you want to implement methods in paper? For instance, joint model training, joint loss etc. Is Pytorch better in this case?

**Comment by sushreebarsa** [comment]
@hockman1 Could you please elaborate your point and in order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thanks!

**Comment by sushreebarsa** [comment]
Closing this issue due to lack of recent activity.Please feel free to reopen if you still have concern.Thanks!
